### PR TITLE
Add game interface

### PR DESCRIPTION
**Pull request by Blaise-Shyaka**
- Describe step by step how the game will work and how the user will interact with it﻿

### CODE REVIEW

**Review by voscarmv — CHANGES_REQUESTED** [review]
# Impressive work, team! 🖥️ 👀 ✨ 
## However, some changes are still required... 🧰 🔨 🔧 

- Please rename your file `bin/main' to 'bin/main.rb', as specified in the Pathwright instructions.

### Submit these changes and request a new code review! 😄 👍 

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Review by euqueme — CHANGES_REQUESTED** [review]
> ### Pull Request reviewed by Maria Eugenia Queme P. (TSE)

### Hello @Blaise-Shyaka :lollipop: 

### Thank you for doing the changes requested by the previous reviewer, :medal_sports: your logic is good I like that you are asking for the player's names this makes your game human friendly and you are using letters instead of numbers for your game, please keep it because makes your game stand out a whole lot from the rest :video_game: :clap: please work on the following changes :construction: :construction_worker: 

##### Implementation details
> I'm sorry this wasn't pointed out by the previous reviewer but this is actually a mandatory requirement, that's why I'm sharing an example of how to do it with code, so let's add it to your code

* [x] A loop is needed in this game (recommended is `while` but anything else will do) ([OdinProject hint](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/ruby/OOP/milestone2/loop.png))
* [x] Inside the loop, you should call actions repeated for each move
* [x] A loop needs to be terminated at some point (after a winning move or draw to be more specific). It can be done with variable:

```ruby
game_on = true

# initial instructions for player

while game_on
  # loop for each move
  
  if winner # and/or draw (the exact condition in this milestone is not important)
    game_on = false
  end
end
```

### That's it! :raised_hands: 

### Please :pray: do the changes :technologist: and submit back for a code review when you are done :memo: 

Happy coding! :rocket: 

Any questions? ask me on slack **@ Maru** :woman_technologist: 

PROJECT STATUS: REQUESTED CHANGES :traffic_light:

File: bin/main.rb [hunk 1]
```diff
@@ -12,28 +12,39 @@
...
 
-"
-puts "#{player1}, Please choose a move by typing one of the letters"
-player1_move = gets.chomp
-# 4. Check if the move is a win, update the board and print the updated board
-puts "You won if #{player1_move} is a winning move"
-puts "
+# 3. Define a dummy winning condition (NOT the real game logic)
+def winner(move)
+    winning_move = 'e'
```
Comment (kobiyoyo) [inline bin/main.rb]: Can a boolean be used for the move and winning_move instead

**Review by kobiyoyo — CHANGES_REQUESTED** [review]
## Hi Team    
#### Great Job so far, everything works fine, but there are a few more things required. 
### General

- [ ] Check the comment below 
### Make changes and submit another code review request.
*Happy coding* 
(Any questions Feel free to contact me on slack @kobiyoyo)

**Review by voscarmv — APPROVED** [review]
# Project status: Approved ✨ 😎👍 💯 ✔️

- All basic requirements seem to have been met here.

### You may merge this branch and move forward ⛙🛣

### Questions or comments? Reach me on [Twitter](https://twitter.com/voscarmv) or [LinkedIn](https://linkedin.com/in/oscar-mier) 🧑‍💻🗨️🧑‍💻 
# Happy Hacking! 🧠 ⌨️ 🖱️ 🖥️

**Pull request merged**